### ISSUE TITLE
Will participants receive a gift?

### ISSUE THREAD

**Issue by jackivan88** [seoul-christian-school/hackathon-one#16]
Will people who come get a gift just for coming? Common ideas are things like t-shirts or hoodies that they can wear to show that they attended an event.

**Comment by brighton818** [comment]
Yes. It can be unified t-shirt that people have to wear for hackathon.

**Comment by jackivan88** [comment]
Will we get their shirt size when they register? Who will handle designing
the T-shirt?
